samtranslator/model/stepfunctions/events.py

```diff
@@ -28,6 +28,8 @@ class EventSource(ResourceMacro):
     # TODO: Make `EventSource` an abstract class and not giving `principal` initial value.
     principal: str = None  # type: ignore
 
+    Target: Optional[Dict[str, str]]
```

Comment:
We need to figure out some way so we could use the classes from the schema directly throughout the code. Not needed in this PR.

Reply:
Yeah that sounds like a good idea.